index.js

```diff
@@ -1,5 +1,8 @@
 let items = [];
 let newDiv;
```

Comment:
new div never used outside of createBlock,  move declaration of it to local scope.